crypto/evp/evp_enc.c

```diff
@@ -440,6 +440,176 @@ static int evp_cipher_init_internal(EVP_CIPHER_CTX *ctx,
     return 1;
 }
 
+/*
+ * FIXME this function has a lot of common with evp_cipher_init_internal
+ * so we need to refactor both of them later
+ */
+static int evp_cipher_init_skey_internal(EVP_CIPHER_CTX *ctx,
+                                         const EVP_CIPHER *cipher,
+                                         const EVP_SKEY *key,
+                                         const unsigned char *iv, int enc,
+                                         const OSSL_PARAM params[])
+{
+    int iv_len = 0;
+    void *pkeydata = (key == NULL) ? NULL : key->keydata;
```

Comment:
As in the previous commit: pkey.* -> skey.*

Reply:
Done, thanks